packages/cmarker/cmarker.0.3.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Pablo Meier <pablo.a.meier@gmail.com>"
+authors: "Jonathan Chan <jyc@fastmail.fm>"
+homepage: "https://github.com/pablo-meier/ocaml-cmark"
+synopsis: "Bindings for a local installation of CMark"
+bug-reports: "https://github.com/pablo-meier/ocaml-cmark/issues"
+license: "BSD 2-Clause"
+dev-repo: "git+https://github.com/pablo-meier/ocaml-cmark.git"
+build: [ "dune" "build" "-p" name]
+depends: [
+  "ctypes" {>= "0.4.1"}
+  "ctypes-foreign" {>= "0.4.0"}
+  "utop" {= "2.4.0"}
```

Comment:
Could you untighten the constraints a little. This seems a little tight for one use of `UTop_main.main`

Reply:
any thoughts on that?